cli/src/lib/libDefs.js

```diff
@@ -64,11 +64,23 @@ async function rebaseCacheRepo(verbose?: VerboseOutput) {
 }
 
 /**
+ * Utility wrapper for ensureCacheRepo with an update expiry of 0 hours.
+ */
+async function updateCacheRepo(verbose?: VerboseOutput): Promise<boolean> {
+  try {
```

Comment:
Added the changes... not sure if I should always output the complete error object (I currently hide this behind the debug flag)... if no debug flag is provided, it will print the `e.message` only...

I personally get confused when a stack-trace is printed whenever I use a commandline-tool (usually tastes like a broken tool)... what do you think?